app/lib/frontend/templates/views/pkg/package_list.dart

```diff
@@ -228,7 +228,7 @@ d.Node _item({
                   image: d.Image(
                     src:
                         staticUrls.getAssetUrl('/static/img/schedule-icon.svg'),
-                    alt: 'icon indicating recent time',
+                    alt: 'new package',
```

Comment:
Maybe:
```suggestion
                    alt: 'recently uploaded package',
```

Reply:
I think "recently created package" may be slightly better, otherwise it could be misread with "recently updated package". Updating...